tests/SConscript

```diff
@@ -0,0 +1,3 @@
+# -*- python -*-
```

Comment:
`obs_base` provides some tests that can be automatically run if you provide some mapping information.  It seems like at least those tests should be run.